include/radio_types.h

```diff
@@ -314,6 +314,22 @@ typedef enum radio_network_scan_specifier_1_5_type {
     RADIO_NETWORK_SCAN_SPECIFIER_1_5_NGRAN
 } RADIO_NETWORK_SCAN_SPECIFIER_1_5_TYPE; /* Since 1.5.2 */
 
+typedef enum radio_frequency_type {
+    RADIO_FREQUENCY_TYPE_RANGE,
+    RADIO_FREQUENCY_TYPE_CHANNEL_NUMBER,
+} RADIO_FREQUENCY_TYPE; /* Since 1.5.4 */;
+
+typedef enum radio_frequency_range {
+    /* Indicates the frequency range is below 1GHz. */
+    RADIO_FREQUENCY_RANGE_LOW = 1,
+    /* Indicates the frequency range is between 1GHz and 3GHz. */
+    RADIO_FREQUENCY_RANGE_MID,
+    /* Indicates the frequency range is between 3GHz and 6GHz. */
+    RADIO_FREQUENCY_RANGE_HIGH,
+    /* Indicates the frequency range is above 6GHz (millimeter wave frequency). */
+    RADIO_FREQUENCY_RANGE_MMWAVE,
+} RADIO_FREQUENCY_RANGE; /* Since 1.5.4 */;
```

Comment:
Please add
```c
G_STATIC_ASSERT(sizeof(RADIO_FREQUENCY_RANGE) == 4);
```
because it's declared as
```c
enum class FrequencyRange : int32_t {
```
in  `types.h` generated by hidl-gen

Reply:
Done